coss/opensource_clubs/templates/opensource_clubs/about_page.html

```diff
@@ -10,45 +10,8 @@
   <img src="{% static 'img/mozilla-clubs-white.svg' %}" alt="mozilla">
 {% endblock %}
 
-{% block extra_about_nav %}
-  <a class="nav-link" href="#roles">Club Roles</a>
-{% endblock %}
-
 {% block extra_about %}
-  <div id="roles" class="roles">
-    <div class="container">
-      <h4>Club Roles</h4>
-
-      <div class="row">
-        <div class="col-md-3">
-          <nav id="nav-roles" class="nav flex-column">
-            <a class="nav-link active" href="#role-captain">Club Captain</a>
-            <a class="nav-link" href="#role-coordinator">Regional Coordinator</a>
-            <a class="nav-link" href="#role-member">Member</a>
-          </nav>
-        </div>
-        <div class="col-md-8 ml-auto well">
-          <div class="nav-content active" id="role-captain">
-            {{ page.roles_captain|richtext }}
-          </div>
-
-          <div class="nav-content" id="role-coordinator">
-            {{ page.roles_coordinator|richtext }}
-          </div>
-
-          <div class="nav-content" id="role-member">
-            {{ page.roles_member|richtext }}
-          </div>
-        </div>
-      </div>
-    </div>
+  <div class="collaborators">
```

Comment:
Since we have the snippet/streamfield for activities, don't we want to add the missing section in this PR?